tools/sense_studio/templates/project_details.html

```diff
@@ -57,6 +57,10 @@ <h4>Annotated Videos</h4>
                             <i class="edit icon"></i>
                             Annotate
                         </a>
+                        <a class="ui icon button" href="{{ url_for('record_video', split='train', label=class, path=path) }}">
+                            <i class="edit icon"></i>
```

Comment:
Should get a different icon. Also the two buttons don't fit that well next to each other.
![image](https://user-images.githubusercontent.com/5707898/106746377-c1b94d80-6622-11eb-8520-7b1d2e8c0c83.png)

Reply:
is there somewhere a list of icons for semantic ui ?

Reply:
Yes, you can easily search here: https://semantic-ui.com/elements/icon.html